### PR TITLE
Enable custom registry

### LINKED ISSUE #167: ghcr.io should be allowed
The GitHub container registry (ghcr.io) allows for anonymous pulls/reads.

Please consider allowing us to use our driver images that are hosted on `ghcr.io`, rather than requiring us to mirror driver images into dockerhub.

### PR DESCRIPTION
**Pull request by BernhardFuchs**
Closes #167

### CODE REVIEW

File: docs/driver-development.md [hunk 1]
```diff
@@ -58,19 +58,19 @@ In your PR, edit the following files in the Universal Resolver root directory:
...
+As DID methods are developed and matured, the Universal Resolver should maintain its DID drivers with new changes. Contributors should keep their drivers up-to-date as changes happen to the DID Core spec and the DID method spec. Contributors may only wish to direct users to the latest driver, or they may have a `stable` version, a `developer` version, etc. The driver version specified in the `README.md` file should be reflected in a Docker image with a tag that matches the driver version.
 
 In order to update a driver, simply submit a new PR that increments the Docker image version and updates the relevant files (see above in the "How to Contribute a Driver" section).
 
 ## How to test a driver locally
 
-Once your driver is implemented and published as a docker container on DockerHub, you may want to test that it is running properly within the Universal Resolver.
+Once your driver is implemented and published as a docker container, you may want to test that it is running properly within the Universal Resolver.
```
Comment (peacekeeper) [inline docs/driver-development.md:73]:
```suggestion
Once your driver is implemented and published as a Docker container, you may want to test that it is running properly within the Universal Resolver.
```

File: docs/driver-development.md [hunk 2]
```diff
@@ -32,15 +32,15 @@ https://github.com/decentralized-identity/universal-resolver/pull/100
...
 
-In order to contribute a driver to the Universal Resolver, the driver's source code must be published. You may choose to publish it at the [Decentralized Identity Foundation](https://github.com/decentralized-identity/universal-resolver/tree/master/drivers) or on another publicly available site. In addition, your Docker image must be published on [DockerHub](https://hub.docker.com/) with version tags to allow configuration of your driver. Ideally, the same DockerHub image is kept up-to-date to preserve versioning history (see below in the "How to Update a Driver" section).
+In order to contribute a driver to the Universal Resolver, the driver's source code must be published. You may choose to publish it at the [Decentralized Identity Foundation](https://github.com/decentralized-identity/universal-resolver/tree/master/drivers) or on another publicly available site. In addition, your Docker image must be published on a publicly accessible container registry with version tags to allow configuration of your driver. Ideally, the same docker image is kept up-to-date to preserve versioning history (see below in the "How to Update a Driver" section).
```
Comment (peacekeeper) [inline docs/driver-development.md:43]:
```suggestion
In order to contribute a driver to the Universal Resolver, the driver's source code must be published. You may choose to publish it at the [Decentralized Identity Foundation](https://github.com/decentralized-identity/universal-resolver/tree/master/drivers) or on another publicly available site. In addition, your Docker image must be published on a publicly accessible container registry with version tags to allow configuration of your driver. Ideally, the same Docker image is kept up-to-date to preserve versioning history (see below in the "How to Update a Driver" section).
```

File: docs/driver-development.md [hunk 3]
```diff
@@ -58,19 +58,19 @@ In your PR, edit the following files in the Universal Resolver root directory:
   * driver version (e.g. `0.1`)
   * DID spec version that the driver conforms to, with link to DID spec
   * DID method spec version (e.g. `0.1`), with link to DID method spec (or mark "missing")
-  * Docker image name (e.g. `exampleorg/uni-resolver-driver-did-example`) with link to Docker image at DockerHub
+  * Docker image name (e.g. `exampleorg/uni-resolver-driver-did-example`) with link to Docker image at docker-registry
```
Comment (peacekeeper) [inline docs/driver-development.md:61]: Hmm will people understand here what is meant by "docker-registry" ?

**Review by peacekeeper — APPROVED** [review]
Looks great. Some suggestions inline.

File: docs/driver-development.md (hunk 3 above)
Comment (BernhardFuchs) [reply docs/driver-development.md:61]: What about `with link to hosted Docker image`

**Pull request merged**